src/classes/Forms.ts

```diff
@@ -324,4 +325,6 @@ export default (tenant: Tenant) =>
     static decryptUserToken(details: { userToken: string; secret: string }) {
       return decryptUserToken(details)
     }
+
+    static getConditionalPredicatesItemSchema = ConditionalPredicatesItemSchema
```

Comment:
The issue with returning the schema and not a function that validates the data is that this schema relies on a specific version of `Joi`. If we export this, we need to ensure we are using the same version and that we dont update the version accidentally in the future. Is there any way we can export a function that validates the data instead?